fm4tlp/tgbl_comment_dataprep.py

```diff
@@ -34,6 +34,8 @@
 import tensorflow.compat.v1 as tf
 import tqdm
 
+if 'gfile' not in sys.modules:
```

Comment:
This block no longer exists in the codebase -- possibly you need to sync with the current commit?

Reply:
it is added in previous iteration, please let us know if there are any concerns.